.github/workflows/create-preview-link.yml

```
@@ -0,0 +1,118 @@
+name: Build Transcripts Preview Site
+
+on:
+  pull_request:
+    paths:
+      - '**' 
+  workflow_dispatch:
+jobs:
+  build-preview:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup Checkout for Bitcoin Transcripts
+        uses: actions/checkout@v4
+        with:
+          path: transcripts
+        
+      - name: Setup Node.js 20
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Clone Registry Repo
+        run: git clone https://github.com/bitcointranscripts/registry.git
+
+      - name: Replace public/bitcoin-transcript in Registry with contents of Bitcoin Transcripts
```

@0tuedon thanks for clarifying what this step does. Does this rephrasing work?

```suggestion
      - name: Replace the public/bitcoin-transcript submodule in Registry with contents of this current branch
```